### PR TITLE
Release Akka 2.6.10

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

**Review by johanandren — APPROVED** [review]

File: blog/_posts/2020-10-09-akka-2.6.10-released.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+There is also a new [health check for Cluster Sharding](https://doc.akka.io/docs/akka/2.6/typed/cluster-sharding.html#health-check) that you can enable when you don’t want to receive production traffic until the local shard region is ready to retrieve locations for shards.
+
+The new rebalance alogorithm can reach optimal balance in a few rebalance rounds (typically 1 or 2 rounds). You can still configure limits of how many shards to move in each round. Compared to the old algorithm that would only move one shard in each round (every 10 seconds). The new algorithm is recommended and will become the default in future versions of Akka, but currently you have to [enable it explicitly as described in the documenation](https://doc.akka.io/docs/akka/2.6/typed/cluster-sharding.html#shard-allocation).
```
Comment (ennru) [inline blog/_posts/2020-10-09-akka-2.6.10-released.md:38]:
```suggestion
The new rebalance algorithm can reach optimal balance in a few rebalance rounds (typically 1 or 2 rounds). You can still configure limits of how many shards to move in each round. Compared to the old algorithm that would only move one shard in each round (every 10 seconds). The new algorithm is recommended and will become the default in future versions of Akka, but currently you have to [enable it explicitly as described in the documenation](https://doc.akka.io/docs/akka/2.6/typed/cluster-sharding.html#shard-allocation).
```

**Review by ennru — APPROVED** [review]
LGTM.

File: blog/_posts/2020-10-09-akka-2.6.10-released.md (hunk 1 above)
Comment (raboof) [inline blog/_posts/2020-10-09-akka-2.6.10-released.md:38]: Another typo ;)

```suggestion
The new rebalance algorithm can reach optimal balance in a few rebalance rounds (typically 1 or 2 rounds). You can still configure limits of how many shards to move in each round. Compared to the old algorithm that would only move one shard in each round (every 10 seconds). The new algorithm is recommended and will become the default in future versions of Akka, but currently you have to [enable it explicitly as described in the documentation](https://doc.akka.io/docs/akka/2.6/typed/cluster-sharding.html#shard-allocation).
```

**Review by raboof — APPROVED** [review]
LGTM after fixing typo

**Pull request merged**